gsoc/gsoc2016/ideas_eras.md

```diff
@@ -0,0 +1,44 @@
+# Italian Mars Society's ERAS Project #
+
+## Sub-Org Details ##
+
+Sub-org admin: Antonio Del Mastro - <gsoc@marssociety.it>
+
+Main repository: <https://bitbucket.org/italianmarssociety/eras/src>
+
+Mailing list: <https://groups.google.com/forum/#!forum/erasproject>
+
```

Comment:
This above information should go to the [members page](http://openastronomy.org/members/), following for example the [Astropy](https://github.com/OpenAstronomy/openastronomy.github.io/blob/master/_includes/members/Astropy.html) page.